### ISSUE TITLE
[MD041] Error with metadata

### ISSUE THREAD

**Issue by Spixmaster** [markdownlint/markdownlint#493]
I use version 0.13.0 and the file content

```markdown
---
title: "Asdf"
author: "My Name"

numbersections: true
toc: true
---

# Asdf
```

results in `MD041/first-line-h1: First line in file should be a top level header`.

I have read the rule https://github.com/DavidAnson/markdownlint/blob/main/doc/md041.md. It is stated that metadata is handled correctly but apparently it is not.

My configuration.
```rb
all
exclude_rule "MD024"
rule "MD007", :indent => 2
rule "MD013", :line_length => 120
```

**Comment by DavidAnson** [comment]
You are referencing the documentation for the Node implementation of markdownlint, but using the Ruby implementation. The Ruby version may not respect the title field in front matter as it is not mentioned in the relevant documentation: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md041---first-line-in-file-should-be-a-top-level-header

**Comment by Spixmaster** [comment]
Thank you very much, @DavidAnson! You are right. I switched to your project and it works. I confused the projects because the rules are named equally.
